browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -372,18 +373,26 @@ void RewardsDOMHandler::GetGrant(const base::ListValue* args) {
 
 void RewardsDOMHandler::OnGrantCaptcha(
     brave_rewards::RewardsService* rewards_service,
-    std::string image,
+    std::string response,
     std::string hint) {
+#if defined(OS_ANDROID)
```

Comment:
this logic doesn't seem to have anything do to with the UI, shouldn't it be in rewards_service_impl or possibly even ledger?